### PR TITLE
Update hardcoded LutMetadata for HCAL TP

### PR DESCRIPTION
**Pull request by matz-e**
- Revert the gain that was changed in the 6XY series back to its original value
- Provide mores sensible RCalib values for running with Phase1 upgrades
- Adjust TDC mask to cover all allowed values for now (missing 4 highest bits)

### CODE REVIEW

**Comment by kpedro88** [comment]
@cmsbuild please test

**Comment by kpedro88** [comment]
@mmusich can you approve this? (Corresponding database conditions have already been updated and approved in #16048...)

**Comment by mmusich** [comment]
+1

File: CalibCalorimetry/HcalPlugins/src/HcalHardcodeCalibrations.cc [hunk 1]
```diff
@@ -591,22 +591,41 @@ std::unique_ptr<HcalLutMetadata> HcalHardcodeCalibrations::produceLutMetadata (c
...
-    HcalLutMetadatum item(cell->rawId(),1.0,1,1);
+  for (const auto& cell: cells) {
+    float rcalib = 1.;
+    int granularity = 1;
+    int threshold = 1;
+
+    if (dbHardcode.useHEUpgrade() or dbHardcode.useHFUpgrade()) {
+       switch (cell.genericSubdet()) {
+          case HcalGenericDetId::HcalGenBarrel:
+             rcalib = 1.128;
```
Comment (davidlange6) [inline CalibCalorimetry/HcalPlugins/src/HcalHardcodeCalibrations.cc]: @matz-e - could you add a few comments to explain where these hardcoded numbers came from? Thanks

Comment (matz-e) [inline CalibCalorimetry/HcalPlugins/src/HcalHardcodeCalibrations.cc]: @davidlange6 yes, I can do that. I'm not sure where the originals (L609 that I removed/changed) came from, but I can at least document the changes.

**Comment by matz-e** [comment]
Documented new hardcoded values…

**Comment by mmusich** [comment]
+1

**Comment by mmusich** [comment]
please test

**Comment by kpedro88** [comment]
looks like tests timed out for some reason

**Comment by kpedro88** [comment]
@cmsbuild please test

**Pull request merged**